_data/simple-icons.json

```
@@ -645,6 +645,11 @@
             "hex": "F94877",
             "source": "https://foursquare.com/about/logos"
         },
+        {
+            "title": "Figma",
+            "hex": "F24E1E",
+            "source": "https://http://figma.com/"
```

Looks like something went wrong here 😛 

```suggestion
            "source": "https://figma.com/"
```